tokio-util/src/codec/framed.rs

```diff
@@ -284,6 +284,7 @@ where
 ///
 /// [`Framed`]: crate::codec::Framed
 #[derive(Debug)]
+#[allow(clippy::manual_non_exhaustive)]
```

Comment:
In general, when adding `allow` attributes, I like to add a comment explaining why that lint is allowed. Not a blocker, but could be nice.